Windows/DocViewer.js

```diff
@@ -138,7 +149,14 @@ define([
              * @param {string} docId - documentation item id
              */
             win.loadDocId = function(docId) {
-                win.loadDocUrl(Module._docRoot + `/${docId}.html`);
+                let url = Module._docRoot + `/${docId}.html`
```

Comment:
Funny that the template literal is used together with a string concatenation :-)

Reply:
Indeed, I didn't notice that, I included the doc root in the template literal.